saleor/graphql/product/bulk_mutations/products.py

```diff
@@ -371,11 +371,25 @@ def perform_mutation(cls, _root, info, ids, **data):
             ).values_list("pk", flat=True)
         )
 
+        product_pks = list(
```

Comment:
I wonder if all this logic from `perform_mutation` should be done atomically?

Reply:
👍  I'll add a transaction there.